.github/workflows/build.yaml

```diff
@@ -26,6 +26,11 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v3
+    - name: Dependency Review
+      uses: actions/dependency-review-action@v4
+      with:
+        vulnerability-check: true
+        license-check: false
```

Comment:
How about `comment-summary-in-pr`? May save some time to open the pipeline and see what actually failed.